pkgs/test/test/runner/precompiled_test.dart

```diff
@@ -235,6 +236,7 @@ Future<Null> _writePackagesFile() async {
   // TODO: remove try/catch when this issue is resolved:
   // https://github.com/dart-lang/package_config/issues/66
   try {
+    await Directory(p.join(d.sandbox, '.dart_tool')).create();
```

Comment:
I think this is more idiomatic as

```suggestion
    await d.dir('.dart_tool').create();
```

Although that also means that `savePackageConfig` should probably be doing something similar... We can do it in a followup.

Reply:
I did file an issue on package_config about this